analyzers/tests/SonarAnalyzer.UnitTest/TestCases/EmptyMethod.Comment.Fixed.cs

```diff
@@ -22,6 +22,24 @@ void F3()
             // Method intentionally left empty.
         }
 
+        public void ConditionalCompilation()
+        {
+#if SomeThing
+            Console.WriteLine();
+#endif
+        }
+
+        public void ConditionalCompilationEmpty() //FP. We don't check if the conditional contains anything.
+        {
+#if SomeThing
+#endif
+        }
+
+        public void EmptyRegionTrivia() // Fixed
+        {
+            // Method intentionally left empty.
```

Comment:
`#region` trivia (and probably other structured trivia) gets deleted by this fixer. Seems to be an unrelated problem to me.